.github/workflows/helm-chart.yml

```diff
@@ -17,12 +17,18 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: 'recursive'
-      - uses: DeterminateSystems/nix-installer-action@v19
-      - name: Pre-populate nix-shell
+      - name: "Extract NIX_PATH"
+        id: nixpath
         run: |
-          export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' nix/sources.json -r)
-          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
-          nix-shell deploy/helm/shell.nix --run "echo"
+          echo "nix_path=nixpkgs=$(jq -r '.nixpkgs.url' nix/sources.json)" >> $GITHUB_OUTPUT
+      - uses: cachix/install-nix-action@v25
+        with:
+          nix_path: "${{ steps.nixpath.outputs.nix_path }}"
+      - name: Setup nix cache
+        uses: cachix/cachix-action@v14
+        with:
+          name: nix-cache
```

Comment:
Doesn't this have to be the name of our cache in cachix?
Also we'd to specify the token in order to write?
Do they even have a free tier for OSS?

Reply:
Yeah, I was thinking that it stores them in GitHub Actions cache, but it is using the cachix itself, it is free for OSS softwares

Reply:
Ah the magic nix cache has now been restored, so we could use it again

Reply:
Actually, do we even need the cache here? We're not building our own derivations

Reply:
No we don't need, I have added this and I was thinking that it stores cache in GitHub Actions